project_task_link/models/common.py

```diff
@@ -0,0 +1,80 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import re
+from lxml import etree
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using etree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace etree with the equivalent defusedxml package.](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4478774)